doc/specs/dna.md

```diff
@@ -156,6 +156,15 @@ They are available in keyword specific Capabilities and function names
             },
             // ...
           ],
+          "evt_declarations": [
```

Comment:
Why not just `events`? It's so much simpler.

Reply:
fixed

Reply:
I still see "evt_declarations". Need to push?
As stated above, I would name this property ```signals```.

Reply:
heh, sorry I fixed it in the adr, not the dna spec